.github/workflows/main.yml

```
@@ -26,6 +26,14 @@ jobs:
           repo: Solana.Unity-SDK
           plugin_contributors: yes
           plugin_contributors_contributions: yes
+            - name: Upload Test Report
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)